discord_bot.py

```diff
@@ -66,7 +66,8 @@ def as_message(message: discord.Message) -> Message:
         author_name=message.author.name,
         author_mention=message.author.mention,
         message_id=message.id,
-        content=message.content
+        content=message.content,
+        is_file=len(message.attachments) > 0
```

Comment:
I assume the file is some kind of URL or resource ID. We should pass that through in the `Message` so we have the option of using it in the future.